selfdrive/locationd/test/test_runner.cc

```diff
@@ -0,0 +1,2 @@
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
```

Comment:
give this file a better name. there's lot of different things in `selfdrive/locationd`